frontend_tests/node_tests/markdown.js

```diff
@@ -758,13 +758,12 @@ run_test("missing unicode emojis", () => {
         // return undefined
     }
 
-    const config = {
-        ...markdown_config.get_helpers(),
-        get_emoji_name: fake_get_emoji_name,
-    };
-
-    markdown.initialize(page_params.realm_filters, config);
+    const actual_get_emoji_name = emoji.get_emoji_name;
+    emoji.get_emoji_name = fake_get_emoji_name;
 
     markdown.apply_markdown(message);
     assert.equal(message.content, "<p>\u{1f6b2}</p>");
+
+    // restore get_emoji_name for any future tests that follow
+    emoji.get_emoji_name = actual_get_emoji_name;
```

Comment:
This works but we should maybe develop a better try/finally pattern for this sort of thing.